### PR TITLE
Pull request to update with editorial updates for weblogic-on-oke

### PR DESCRIPTION
**Pull request by coltse**
@streamnsight - These are outstanding editorial changes as identified by the OHC Learn Publishing Team.

### CODE REVIEW

File: developer-library/weblogic-to-oci/on-prems-env-docker/on-prems-env-docker.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ To run this lab, you will need:
 
 ### **On Linux**
 
-*Make sure you followed the installation instructions to set up Docker as root and:*
+Make sure you followed the installation instructions to set up Docker as root and:
```
Comment (streamnsight) [inline developer-library/weblogic-to-oci/on-prems-env-docker/on-prems-env-docker.md:44]: I think that's the only place I would keep the emphasis. I see other places it was kept so it looks good.

Comment (coltse) [reply developer-library/weblogic-to-oci/on-prems-env-docker/on-prems-env-docker.md]: I've added back the emphasis

File: developer-library/weblogic-to-oci/on-prems-env-docker/on-prems-env-docker.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ To run this lab, you will need:
     ```bash
     <copy>
     cd ~/
-    wget https://objectstorage.us-ashburn-1.oraclecloud.com/p/ufHAcuj6M4xpPsqYVmHBsSA7vFUiAsx7GF_gZMB9hyo/n/ocisateam/b/images/o/weblogic-to-oci.zip
+    wget https://objectstorage.us-ashburn-1.oraclecloud.com/p/ufHAcuj6M4xpPsqYVmHBsSA7vFUiAsx7GF_gZMB9hyo/n/ocisateam/b/images/o/weblogic-to-oci.zip.
```
[the remark this answers is not in the document]
Comment (coltse) [reply developer-library/weblogic-to-oci/on-prems-env-docker/on-prems-env-docker.md]: I've removed the dot

**Comment by streamnsight** [comment]
@coltse Looks good to me now.
I'll work on the long log file issue.

**Review by tmcginn — APPROVED** [review]
Colleen,
Going forward, please include the WMS ID for the workshop(s) in your pull request (bit.ly/oraclewms).
For example, this one is WMS ID: 3380.
Thanks, Tom

**Comment by coltse** [comment]
> Colleen,
> Going forward, please include the WMS ID for the workshop(s) in your pull request (bit.ly/oraclewms).
> For example, this one is WMS ID: 3380.
> Thanks, Tom

Hi Tom, will do going forward, thanks!

**Pull request merged**